src/main/content/index.html

```diff
@@ -162,26 +162,52 @@ <h2 class="develop_title">Monitor microservices in the cloud</h2>
                 <h3 id="logos_title">Open Liberty integrates with the Open Source community</h3>
                 <div id="logo_image_row" class="row justify-content-center">
                     <div class="col-sm-12 col-md">
-                        <img id="microprofile_logo"  src="/img/home_logo_microprofile.png" alt="MicroProfile Logo">
+                        <a href="https://microprofile.io/">
+                            <img id="microprofile_logo"  src="/img/home_logo_microprofile.png" alt="MicroProfile Logo">
+                        </a>
                     </div>
                     <div class="col-sm-12 col-md">
-                        <img id="jakarta_ee_logo"  src="/img/home_logo_jakartaee.png" alt="Jakarta EE logo">
+                        <a href="https://jakarta.ee/">
+                            <img id="jakarta_ee_logo"  src="/img/home_logo_jakartaee.png" alt="Jakarta EE logo">
+                        </a>
                     </div>
                 </div>
                 <div class="logos_row row justify-content-center">
-                    <p>Jenkins</p>
-                    <p>Spring</p>
-                    <p>Docker</p>
-                    <p>Grafana</p>
-                    <p>Gradle</p>
-                    <p>Logstash</p>
+                    <a href="https://www.jenkins.io/">
+                        <p>Jenkins</p>
+                    </a>
+                    <a href="https://spring.io/">
+                        <p>Spring</p>
+                    </a>
+                    <a href="https://www.docker.com/">
+                        <p>Docker</p>
+                    </a>
+                    <a href="https://grafana.com/">
+                        <p>Grafana</p>
+                    </a>
+                    <a href="https://gradle.org/">
+                        <p>Gradle</p>
+                    </a>
+                    <a href="https://www.elastic.co/logstash/">
+                        <p>Logstash</p>
+                    </a>
                 </div>
                 <div class="logos_row row justify-content-center">
-                    <p>Prometheus</p>
-                    <p>Maven</p>
-                    <p>Kubernetes</p>
-                    <p>Kibana</p>
-                    <p>Elastic Search</p>
+                    <a href="https://prometheus.io/">
+                        <p>Prometheus</p>
+                    </a>
+                    <a href=" https://maven.apache.org/">
```

Comment:
Random space before the URL

Reply:
Resolved in commit https://github.com/OpenLiberty/openliberty.io/pull/2694/commits/4be746698399c7bac0d0ec0d5097ddbb7d3087ed